packages/server/modules/blobstorage/index.ts

```diff
@@ -86,7 +87,12 @@ const ensureConditions = async () => {
   }
 }
 
-const errorHandler = async (req, res, callback) => {
+type ErrorHandler = (
+  req: Request,
+  res: Response,
+  callback: (req: Request, res: Response) => Promise<void>
+) => Promise<void>
+const errorHandler: ErrorHandler = async (req, res, callback) => {
```

Comment:
there's both a RequestHandler and ErrorRequestHandler (w/ 4th err param) types in `express`

Reply:
i see, never mind then